src/processor/containerprocessor/log_constructor.go

```diff
@@ -0,0 +1,77 @@
+package containerprocessor
+
+import (
+	"go.opentelemetry.io/collector/pdata/pcommon"
+	"go.opentelemetry.io/collector/pdata/plog"
+	"time"
+)
+
+const (
+	k8sLogType = "sw.k8s.log.type"
+
+	// Attributes for OTel entity events identification
+	otelEntityEventAsLog = "otel.entity.event_as_log"
+	otelEntityEventType  = "otel.entity.event.type"
+	swEntityType         = "otel.entity.type"
+
+	// Attributes for telemetry mapping
+	otelEntityId     = "otel.entity.id"
+	k8sContainerName = "k8s.container.name"
+	k8sNamespaceName = "k8s.namespace.name"
+	k8sPodName       = "k8s.pod.name"
```

Comment:
There is `semcov` package in opentelemetry which already includes many of these, so you don't need to define your constants. See https://pkg.go.dev/go.opentelemetry.io/collector/semconv@v0.118.0/v1.27.0 

So you simply import `semconv "go.opentelemetry.io/collector/semconv/v1.27.0"` and then you can use `semconv.AttributeK8SNamespaceName` etc